routes/post.py

```diff
@@ -0,0 +1,34 @@
+from app import app, mongo
+from flask_restful import Resource, Api
+from flask import jsonify, request
+from bson.objectid import ObjectId
+
+api = Api(app)
+
+class AddPost(Resource):
+  def post(self):
```

Comment:
this is what defines it as a post? if I replaced with get would it be a get request?

Reply:
yes, if you change the post on line 9 to get in will change it to a get request.